### ISSUE TITLE
Generate Event Handler

### ISSUE THREAD

**Issue by ichsananam** [prominic/Moonshine-IDE#486]
Please enhance IDE with event handler generator (like Ctrl+Space in Flex Builder).

**Comment by piotrzarzycki21** [comment]
@joshtynjala Have you thought about adding support for that ? If I correctly understand it would part of `Code Actions`

**Comment by joshtynjala** [comment]
Yeah, I've considered it, but I haven't spent the time to figure out how to implement it yet. To make it work like Flash Builder, I'd need to detect when completion is requested inside an addEventListener() function call. Then I'd also need to calculate an appropriate place to insert the listener function. It's just non-trivial enough that I can't fit it in quickly when I have a few minutes. It's on my to-do list, though.
